_docs_integrate/delete-an-attribute.md

```diff
@@ -0,0 +1,135 @@
+  ]
+}
+```
+
+Of course, it is also possible to request the deletion of multiple peer shared Attributes within a single Request.
+For this purpose, several DeleteAttributeRequestItems or suitable [RequestItemGroups]({% link _docs_integrate/data-model-overview.md %}#requestitemgroup) can be inserted into the `items` property of the Request.
+
+Before sending the Request, we recommend to [validate its content]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}), since this will give you additional information in case of an error.
+{: .notice--info}
+
+Next, send the Request to the peer.
+You can either do so by [Message]({% link _docs_integrate/data-model-overview.md %}#message) or by a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate), using the `onExistingRelationship` property of a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
+For a detailed explanation check out our guides on how to send [Requests via Messages]({% link _docs_integrate/requests-over-messages.md %}) and [Requests via RelationshipTemplates]({% link _docs_integrate/requests-over-templates.md %}).
+
+Once the peer received the Request, they can accept or reject it.
+If they want to [accept it]({% link _docs_use-cases/use-case-consumption-accept-incoming-request.md %}), they must use the [AcceptDeleteAttributeRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#acceptdeleteattributerequestitemparameters).
+Doing so, they specify a `deletionDate` on which they plan to delete the peer shared Attribute.
+In the given example, the payload would look like the following:
+
+```json
+{
+  "items": [
+    {
+      "accept": true,
+      "deletionDate": "<date the peer shared Attribute will be deleted>"
+    }
+  ]
+}
+```
+
+Now, the [enmeshed Runtime]({% link _docs_explore/61-runtime.md %}) sets the `deletionInfo` of the corresponding peer shared Attribute of the peer with `deletionStatus` `"ToBeDeleted"` and the specified `deletionDate`.
+The same is done for all predecessors of the peer shared Attribute.
+Then, the appropriate [DeleteAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#deleteattributeacceptresponseitem) is generated and sent back in the [Reponse]({% link _docs_integrate/data-model-overview.md %}#response) to the Sender of the Request.
+There, the `deletionInfo` of the corresponding own shared Attribute and its predecessors is set with `deletionStatus` `"ToBeDeletedByPeer"` and the `deletionDate` received in the Response.
+
+It is also possible for the peer to reject the DeleteAttributeRequestItem, if its `mustBeAccepted` property is set `false`, or to [reject the Request]({% link _docs_use-cases/use-case-consumption-reject-incoming-request.md %}) for deleting a peer shared Attribute as a whole, if they have a valid reason for keeping the respective peer shared Attribute.
```

Comment:
- Also mention that the RejectRequestItemParameters should be used to reject the DeleteAttributeRequestItem (if its `mustBeAccepted` property is set `false`), so the required property `accept` with value `"false"` should be used.
- Use consistently plural or singular for 'peer shared Attributes'/'peer shared Attribute' (here, 'peer shared Attribute' is used, and in the next sentence, 'peer shared Attributes').